review/fileToReview.js

```diff
@@ -0,0 +1,73 @@
+
+var domain = "bank.local.fr"
+
+/**
+ * @description Fetch transactions recursively
+ * @param {string} fromDate The maximum date of transactions to return
+ * @param {string} authorization Authorization header to authent the user
+ * @param {jws} jws Jws token, mandatory if we get one from login request
+ * @param {Number} id Account id
```

Comment:
`accountId` would probably be a more meaningful name here
```suggestion
 * @param {Number} accountId User account id
```